Documentation/Books/Cookbook/Monitoring/Collectd.md

```diff
@@ -61,138 +61,218 @@ For the RRD we will go pretty much with defaults:
 
 ### cURL JSON
 
-`Collectd` comes with a wide range of metric aggregation plug-ins. Many tools today use [JSON](http://json.org) as data formating grammar; so does ArangoDB. Therefore a plug-in offering to fetch JSON documents via HTTP is the perfect match as an integration interface:
+`Collectd` comes with a wide range of metric aggregation plug-ins.
+Many tools today use [JSON](http://json.org) as data formating grammar; so does ArangoDB.
+
+Therefore a plug-in offering to fetch JSON documents via HTTP is the perfect match to query ArangoDBs [administrative Statistics interface](../../HTTP/AdministrationAndMonitoring/#read-the-statistics):
 
     # Load the plug-in:
     LoadPlugin curl_json
     # we need to use our own types to generate individual names for our gauges:
-    TypesDB "/etc/collectd/collectd.conf.d/arangodb_types.db"
+    # TypesDB "/etc/collectd/collectd.conf.d/arangodb_types.db"
```

Comment:
the types db residing in `collectd.conf.d` isn't optimal (at least with the current collectd versions), collectd warns about non-`.conf` files there.